internal/services/aws_reservation_service.go

```diff
@@ -118,7 +118,7 @@ func CreateAWSReservation(w http.ResponseWriter, r *http.Request) {
 
 	logger.Debug().Msgf("Enqueuing launch instance job for source %d", reservation.SourceID)
 	launchJob := dejq.PendingJob{
-		Type: jobs.TypeLaunchInstanceAws,
+		Type: queue.TypeLaunchInstanceAws,
```

Comment:
It is not supposed to be dejqQueue now?

Reply:
Services must only use the interface, that is what's defined in `queue_interface.go` nothing else.